conf/application.conf

```diff
@@ -73,6 +73,11 @@ microservice {
          host = "http://localhost:8706"
       }
 
+      soft-drinks-industry-levy-account-frontend {
+         host     = localhost
+         port     = 8707
+      }
+
```

Comment:
have the appConfig files for the environments been updated to include the account-frontend?